.github/workflows/build-and-test.yml

```diff
@@ -15,37 +15,33 @@ jobs:
       uses: actions/checkout@v2.4.0
 
     - name: Install nix
-      uses: cachix/install-nix-action@v15
+      uses: cachix/install-nix-action@v18
```

Comment:
We could consider adding dependabot there.

Reply:
Done in [efeb3f9](https://github.com/tweag/pirouette/pull/163/commits/efeb3f9e6e0dccacd49501f8de3260485ac637b3).